tokio/Cargo.toml

```diff
@@ -94,6 +94,7 @@ pin-project-lite = "0.2.11"
 bytes = { version = "1.2.1", optional = true }
 mio = { version = "1.0.1", optional = true, default-features = false }
 parking_lot = { version = "0.12.0", optional = true }
+unty = "0.0.3"
```

Comment:
The bar for adding new dependencies to Tokio is very high. We will not add this dependency.

Reply:
We can vendor it. The code is very small anyways.

Reply:
The Rust project has previously told me that they don't want Tokio to rely on hacks to do specialization because they are bugs that they want to fix.

Which hack does this crate rely on?

Reply:
This one https://github.com/bincode-org/bincode/issues/665